WooCommerce/Classes/ViewRelated/Coupons/Add and Edit Coupons/CouponExpiryDateView.swift

```diff
@@ -16,31 +18,59 @@ struct CouponExpiryDateView: View {
                     DatePicker("Date picker", selection: $date, displayedComponents: .date)
                         .environment(\.timeZone, timezone)
                         .datePickerStyle(GraphicalDatePickerStyle())
-                        .onChange(of: date) { newDate in
-                            completion(newDate)
-                        }
+                        .onChange(of: date, perform: { newDate in
+                                isRemovalEnabled = true
+                            })
                     Spacer()
+                    VStack {
+                        Divider()
+                        Button(action: {
+                            onCompletion(nil)
+                            presentationMode.wrappedValue.dismiss()
+                        }, label: { Text(Localization.removeButton) })
+                        .buttonStyle(PlainButtonStyle())
+                        .padding(.vertical, Constants.margin)
+                        .foregroundColor(Color(.error))
```

Comment:
Not sure if this would be proper usage of the app-defined colors 😅, but I also didn't find a resource that would reflect the red we're looking for the remove button.